CHANGES.md

```diff
@@ -240,19 +240,20 @@ UUID from the `linkerd-config` ConfigMap!
   * The `stat` command now always shows the number of open TCP connections
   * **Breaking Change** Removed the `--disable-external-profiles` flag from the
     `install` command; external profiles are now disabled by default and can be
-    enabled with the new `--enable-external-profiles` flag
+     enabled with the new `--enable-external-profiles` flag
 * Controller
   * The auto-inject admission controller webhook is updated to watch pods
-    creation and update events; with this change, proxy auto-injection now works
-    for all kinds of workloads, including StatefulSets, DaemonSets, Jobs, etc
+    creation and update events; with this change, proxy auto-injection now
```

Comment:
i think this was already wrapped to 80?

Reply:
I think the the word `works` was just over 80 chars and so I moved that down to the next line which caused the other lines to be formatted.